packages/solarwinds-apm/src/appoptics/exporters/metrics.ts

```diff
@@ -0,0 +1,282 @@
+/*
+Copyright 2023-2024 SolarWinds Worldwide, LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import util from "node:util"
+
+import { type Attributes, type DiagLogger } from "@opentelemetry/api"
+import {
+  type ExportResult,
+  ExportResultCode,
+  type InstrumentationScope,
+} from "@opentelemetry/core"
+import {
+  Aggregation,
+  AggregationTemporality,
+  DataPointType,
+  type ExponentialHistogram as ExponentialHistogramDatapoint,
+  ExponentialHistogramAggregation,
+  type Histogram as HistogramDatapoint,
+  InstrumentType,
+  type MetricDescriptor,
+  type ResourceMetrics,
+} from "@opentelemetry/sdk-metrics"
+import { oboe } from "@solarwinds-apm/bindings"
+import {
+  explicit,
+  exponential,
+  type Histogram,
+} from "@solarwinds-apm/histogram"
+
+export class AppopticsMetricExporter {
+  readonly #reporter: oboe.Reporter
+
+  constructor(
+    reporter: oboe.Reporter,
+    protected readonly logger: DiagLogger,
+  ) {
+    this.#reporter = reporter
+  }
+
+  export(
+    metrics: ResourceMetrics,
+    resultCallback: (result: ExportResult) => void,
+  ): void {
+    for (const scopeMetric of metrics.scopeMetrics) {
+      const scopeTags = this.#scopeTags(scopeMetric.scope)
+
+      for (const metric of scopeMetric.metrics) {
+        const name = `trace.node.${metric.descriptor.name}`
```

Comment:
This means OTel-originated metrics that get exported via apm-proto to AO will all be prefixed with `trace.node.*` (similar to runtime metrics)?